<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Heli</class>
 <widget class="QWidget" name="Heli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1" columnminimumwidth="100,0">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="1">
    <widget class="QComboBox" name="swashType">
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>120</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>120X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>140</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>90</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QGridLayout" name="gridLayout_21">
     <item row="0" column="0">
      <widget class="QCheckBox" name="swashElevatorInvert">
       <property name="text">
        <string>Invert Elevator</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="swashAileronInvert">
       <property name="text">
        <string>Invert Aileron</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="swashCollectiveInvert">
       <property name="text">
        <string>Invert Collective</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="elevatorLabel">
     <property name="text">
      <string>Long. cyc</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="swashRingVal">
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="invertLabel">
     <property name="text">
      <string>Invert</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer_21">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_107">
     <property name="text">
      <string>Swash Ring</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_103">
     <property name="text">
      <string>Swash Type</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="elevatorLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="swashElevatorSource"/>
     </item>
     <item>
      <widget class="QSpinBox" name="swashElevatorWeight">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="aileronLabel">
     <property name="text">
      <string>Lateral cyc</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="aileronLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="swashAileronSource"/>
     </item>
     <item>
      <widget class="QSpinBox" name="swashAileronWeight">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="collectiveLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="swashCollectiveSource"/>
     </item>
     <item>
      <widget class="QSpinBox" name="swashCollectiveWeight">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="collectiveLabel">
     <property name="text">
      <string>Collective</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
